mitmproxy/builtins/replace.py

```diff
@@ -29,16 +31,30 @@ def configure(self, options, updated):
                 raise exceptions.OptionsError(
                     "Invalid regular expression: %s - %s" % (rex, str(e))
                 )
-            lst.append((rex, s, cpatt))
+            if s[0:7] == "file://":
+                file_repl = True
+                if not os.path.isfile(s[7:]):
+                    raise exceptions.OptionsError(
+                        "File not found : %s" % s[7:]
+                    )
+                else:
+                    s = s[7:]
+            lst.append((rex, s, cpatt, file_repl))
         self.lst = lst
 
     def execute(self, f):
-        for rex, s, cpatt in self.lst:
+        for rex, s, cpatt, file_repl in self.lst:
             if cpatt(f):
-                if f.response:
-                    f.response.replace(rex, s)
+                if not file_repl:
```

Comment:
Please flip the two if-else blocks, so we do not need the negation in the condition.